pkgs/development/python-modules/opentelemetry-instrumentation-sqlalchemy/default.nix

```
@@ -0,0 +1,52 @@
+{
+  buildPythonPackage,
+  pythonOlder,
+
+  # build-system
+  hatchling,
+
+  # dependencies
+  opentelemetry-api,
+  opentelemetry-instrumentation,
+  opentelemetry-semantic-conventions,
+  packaging,
+  sqlalchemy,
+  wrapt,
+
+  # tests
+  opentelemetry-test-utils,
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  inherit (opentelemetry-instrumentation) version src;
+  pname = "opentelemetry-instrumentation-sqlalchemy";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  sourceRoot = "${opentelemetry-instrumentation.src.name}/instrumentation/opentelemetry-instrumentation-sqlalchemy";
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    opentelemetry-api
+    opentelemetry-instrumentation
+    opentelemetry-semantic-conventions
+    packaging
+    sqlalchemy
+    wrapt
+  ];
+
+  nativeCheckInputs = [
+    opentelemetry-test-utils
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "opentelemetry.instrumentation.sqlalchemy" ];
+
+  meta = opentelemetry-instrumentation.meta // {
+    homepage = "https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-sqlalchemy";
+    description = "sqlalchemy instrumentation for OpenTelemetry";
```

```suggestion
    description = "SQLAlchemy instrumentation for OpenTelemetry";
```
to comply with style guide for meta.description